### DISCUSSION TITLE
【功能建议】小米插座不支持统计

### DISCUSSION THREAD

**Discussion by adogshz** [XiaoMi/ha_xiaomi_home#306] [Ideas]
小米插座只有瞬时功耗和温度还算靠谱，另外没有电量统计实体，无法跟踪用电情况。麻烦支持组排一下todo。万千小米插用户感谢啊。

**Comment by JimmyKmi** [comment]
请上报model，似乎数据没有通过spec发送，故无法获取

**Reply by adogshz** [reply]
型号为：小米智能插座3，固件版本为：1.1.0.0018；米家智能插线板，固件版本为：1.2.4_58，感谢！

**Reply by JimmyKmi** [reply]
没有统计spec上报，不支持
https://mijia.wiki/products/cuco.plug.v3

下次同类问题可以先去百科检索下，参考“完整自动化”的部分

**Reply by adogshz** [reply]
收到，感谢支持组回复！

**Comment by bjx8970** [comment]
可以通过插座的功率来设置积分传感器的辅助元素来实现用电量统计，目前用了两天，数据跟米家的统计基本一致，我是智能插座3

**Reply by adogshz** [reply]
如何配置积分传感器的辅助啊， 初次使用，还望告知，或者给个提示连结，感谢！

**Reply by bjx8970** [reply]
> 如何配置积分传感器的辅助啊， 初次使用，还望告知，或者给个提示连结，感谢！

home assistant的官方有给示例，在https://www.home-assistant.io/integrations/integration/ 中的Energy（能源）项下面

**Reply by adogshz** [reply]
已经，搞定，经过测试，基本和米家里的统计结果差不多。感谢，感谢！哈哈哈哈，哈哈哈哈
